File: Uri1012.java
import java.util.Scanner;

public class Uri1012 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double c = sc.nextDouble();

        System.out.printf("TRIANGULO: %.3f%n", (a*c)/2);
        System.out.printf("CIRCULO: %.3f%n", 3.14159 * (Math.pow(c, 2)));
        System.out.printf("TRAPEZIO: %.3f%n", ((a+b)*c)/2);
        System.out.printf("QUADRADO: %.3f%n", b*b);
        System.out.printf("RETANGULO: %.3f%n", a*b);
    }
}

File: Uri1012.py
values = list(map(float, input().split()))
A, B, C = values[0], values[1], values[2]

print("TRIANGULO: {:.3f}".format((A * C)/2))
print("CIRCULO: {:.3f}".format(3.14159 * C ** 2))
print("TRAPEZIO: {:.3f}".format(((A + B) * C)/2))
print("QUADRADO: {:.3f}".format(B ** 2))
print("RETANGULO: {:.3f}".format(A * B))